lib/nds_explore.rb

```diff
@@ -0,0 +1,18 @@
+require 'directors_database'
+
+def pretty_print_nds
+  nds = directors_database
+  pp nds
+  nil
```

Comment:
What's the motivation on keeping the `nil`? I put that there originally to signal "hey you need to change this line of code."